app/assets/stylesheets/application.scss

```diff
@@ -751,3 +751,75 @@ nav.breadcrumb {
      font-family: $font-condensed;
 
   }
+
```

Comment:
Is `!important` truly necessary here? Ignore this comment if you think class precedence isn't enough to override these styles.

Reply:
Removed all the `!important` markers.